types/cheesy-api/.eslintrc.json

```diff
@@ -0,0 +1,3 @@
+{
+	"rules": {}
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/cheesy-api/.eslintrc.json;
  Line: 2;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.

Reply:
Please run `prettier` task on this directory:
https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes